test/integration/components/beyla/Dockerfile_dbg

```diff
@@ -26,7 +26,7 @@ COPY Makefile Makefile
 # Build
 RUN make compile-for-coverage
 
-FROM ubuntu:latest
+FROM debian:bookworm-slim
```

Comment:
Shall we use it also for prod?

Reply:
Good idea! I'll make another PR and run some vulnerability scans.